coordinator/src/main/java/org/apache/uniffle/coordinator/SelectStorageStrategy.java

```diff
@@ -17,24 +17,17 @@
 
 package org.apache.uniffle.coordinator;
 
+import java.util.List;
 import java.util.Map;
 
-import org.apache.uniffle.common.RemoteStorageInfo;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
 import org.apache.uniffle.coordinator.LowestIOSampleCostSelectStorageStrategy.RankValue;
 
 public interface SelectStorageStrategy {
 
-  RemoteStorageInfo pickRemoteStorage(String appId);
-
-  void incRemoteStorageCounter(String remoteStoragePath);
-
-  void decRemoteStorageCounter(String storagePath);
-
-  void removePathFromCounter(String storagePath);
-
-  Map<String, RemoteStorageInfo> getAvailableRemoteStorageInfo();
-
-  Map<String, RemoteStorageInfo> getAppIdToRemoteStorageInfo();
+  List<Map.Entry<String, RankValue>> sortPathByRankValue(String path, Path testPath, long time, boolean isHealthy);
 
-  Map<String, RankValue> getRemoteStoragePathRankValue();
+  void setFs(FileSystem fs);
```

Comment:
We would better not assume our remote storage is only FileSystem.

Reply:
Could you elaborate more, or give me some other examples ?

Reply:
If we want to support S3 , COS, Redis or some DBs, it's not suitable for us.

Reply:
Maybe I can remove this method, thinking that if I want to support redis or other databases, I need to re implement the `sortPathByRankValue` method. It's a huge job.